pumps/mongo.go

```diff
@@ -164,9 +165,14 @@ func mongoType(session *mgo.Session) MongoType {
 
 	if result.Code == 303 {
 		return AWSDocumentDB
-	} else {
-		return StandardMongo
 	}
+
+	if result.Code == 115 {
+		return CosmosDB
+	}
```

Comment:
@mativm02 lets convert it into a switch statement, and also what are your toughts about:

```
const(
     AWSDBError = 303
     CosmosDBError = 115
)

switch result.code {
   case AWSError:
      //blahblahblah
      case CosmosDBError:
      // blahblahblah
      Default:
      // StandardMongo
}
```